melosys-eessi-test/src/test/java/no/nav/melosys/eessi/ComponentTestBase.java

```diff
@@ -52,13 +52,17 @@
 @ActiveProfiles(profiles = "test")
 @SpringBootTest(classes = {ComponentTestConfig.class, KafkaTestConfig.class}, webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 @TestPropertySource(locations = "/kafka-test.properties")
+@EmbeddedKafka(controlledShutdown = true, partitions = 1, topics = {"eessibasis-sedmottatt-v1", "eessibasis-sedsendt-v1",
+    "oppgave-endret", "teammelosys.eessi.v1-local"}, brokerProperties = {"offsets.topic.replication.factor=1",
+    "transaction.state.log.replication.factor=1", "transaction.state.log.min.isr=1"}
+)
+@EnableMockOAuth2Server
```

Comment:
Kan vi gjøre noe lurt her og få disse inn i konstanter?

Reply:
Er "disse" properties fra `@EmbeddedKafka`? Hva er fordelen med konstanter her?

Reply:
Redusere skrivefeil når vi bruker disse flere steder i koden

Reply:
Det gjelder bare topics da. Resten skal ikke brukes flere steder i koden. Det er mulig å opprette konstanter i klassen f.eks.

Reply:
Har opprettet konstanter for topics